src/elements/content-sidebar/activity-feed/activity-feed/__tests__/ActivityThreadReplies.test.js

```diff
@@ -0,0 +1,24 @@
+// @flow
+
+import React from 'react';
+import { render } from '@testing-library/react';
+import ActivityThreadReplies from '../ActivityThreadReplies.js';
+import { replies } from '../fixtures';
+
+describe('src/elements/content-sidebar/activity-feed/activity-feed/ActivityThreadReplies', () => {
+    const getWrapper = props => render(<ActivityThreadReplies replies={replies} isExpanded={false} {...props} />);
+
+    test('should render last reply by dafault', () => {
+        const { queryByText } = getWrapper();
+
+        expect(queryByText('Last reply')).toBeVisible();
+        expect(queryByText('Firsy reply')).not.toBeInTheDocument();
```

Comment:
minor: `Firsy reply` -> `First reply`

Reply:
Should we consider grabbing the text from the `replies` fixture directly to avoid such typos?